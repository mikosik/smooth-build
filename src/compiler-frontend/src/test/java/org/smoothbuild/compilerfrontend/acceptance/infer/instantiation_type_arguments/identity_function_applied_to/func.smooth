A myIdentity(A a) = a;
String myFunc(Blob param) = "abc";
myValue = myIdentity(myFunc);
