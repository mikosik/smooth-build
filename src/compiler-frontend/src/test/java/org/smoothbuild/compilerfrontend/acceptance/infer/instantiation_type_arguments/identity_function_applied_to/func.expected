A myIdentity<A>(A a)
  = a;
String myFunc<>(Blob param)
  = "abc";
(Blob)->String myValue<>
  = myIdentity<(Blob)->String>(myFunc<>);
